frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -240,20 +240,21 @@ const handleSubmit = async () => {
                             ? item?.forest_client_number
                             : ''
                     );
+                } else if (error.response.data.detail.code === "self_grant_prohibited") {
+                    errorNotification.msg = 'Granting roles to self is not allowed.';
                 } else {
-                    errorForestClientIdList.push(
+                    errorNotification.errorForestClientIdList.push(
                         item?.forest_client_number
-                            ? item.forest_client_number
-                            : ''
-                    );
+                            ? item?.forest_client_number
+                            : '');
```

Comment:
so here it can be `item?.forest_client_number || ''`, maybe that can help resolve the code smell

Reply:
Hello, Catherine! What code smell is this related to? Thanks for the help!